src/daft-scan/src/file_format.rs

```diff
@@ -116,6 +116,9 @@ pub struct CsvSourceConfig {
     pub delimiter: String,
     pub has_headers: bool,
     pub double_quote: bool,
+    pub quote: char,
+    pub escape_char: char,
+    pub comment: char,
```

Comment:
I think that we should preserve these as `Option` types, as mentioned above.
```suggestion
    pub escape_char: Option<char>,
    pub comment: Option<char>,
```